contracts/external/oracles/DaiPriceOracle.sol

```diff
@@ -280,20 +265,25 @@ contract DaiPriceOracle is
     }
 
     /**
-     * Get the USD price of DAI according to Uniswap given the USD price of ETH.
+     * Get the DAI-USDC price according to Uniswap.
      *
-     * @param  ethUsd  Monetary.Price struct with a fixed-point value with 18 decimals of precision.
-     * @return         Monetary.Price struct with a fixed-point value with 18 decimals of precision.
+     * @return  Monetary.Price struct with a fixed-point value with 18 decimals of precision.
      */
-    function getUniswapPrice(
-        Monetary.Price memory ethUsd
-    )
+    function getUniswapPrice()
         public
         view
         returns (Monetary.Price memory)
     {
-        (uint256 daiAmt, uint256 ethAmt, ) = UNISWAP.getReserves();
-        uint256 uniswapPrice = Math.getPartial(ethUsd.value, ethAmt, daiAmt);
+        (uint256 daiAmt, uint256 poolOneEthAmt, ) = UNISWAP_DAI_ETH.getReserves();
+        (uint256 usdcAmt, uint256 poolTwoEthAmt, ) = UNISWAP_USDC_ETH.getReserves();
```

Comment:
Maybe comment that the order is dependent on the pool. For a different address paired with ETH, the order might not be ETH-2nd